Example/CryptoOnramp Example/CryptoOnramp Example/CryptoOnrampFlowCoordinator.swift

```diff
@@ -17,39 +17,33 @@ final class CryptoOnrampFlowCoordinator: ObservableObject {
 
     /// Represents the possible steps in the flow.
     enum Route: Hashable {
-        case registration
+        case registration(email: String, oAuthScopes: [OAuthScopes])
         case kycInfo
         case identity
-        case authenticated
+        case wallets(customerId: String)
+        case authenticated(customerId: String, wallet: CustomerWalletsResponse.Wallet)
```

Comment:
Rather than storing more optional state within the class, I‘ve migrated to using non-optional associated values.